pkgs/by-name/kc/kcl/package.nix

```diff
@@ -1,44 +1,59 @@
 { lib
+, stdenv
 , buildGoModule
 , fetchFromGitHub
 , kclvm_cli
 , kclvm
 , makeWrapper
 , installShellFiles
+, darwin
 ,
 }:
 buildGoModule rec {
   pname = "kcl";
-  version = "0.8.9";
+  version = "0.9.3";
 
   src = fetchFromGitHub {
     owner = "kcl-lang";
     repo = "cli";
     rev = "v${version}";
-    hash = "sha256-slU3n7YCV5VfvXArzlcITb9epdu/gyXlAWq9KLjGdJA=";
+    hash = "sha256-QUVTRlzG8hT+iQx5dSycbRDAyeknjwGOWynCRw3oxlo=";
   };
 
-  vendorHash = "sha256-Xv8Tfq9Kb1xGFCWZQwBFDX9xZW9j99td/DUb7jBtkpE=";
+  vendorHash = "sha256-AP1MOlnoTnD7luNR+1QtAdMiJL8tEEwJocT+9wBRgAo=";
+
+  # By default, libs and bins are stripped. KCL will crash on darwin if they are.
+  dontStrip = stdenv.isDarwin;
 
   ldflags = [
     "-w -s"
     "-X=kcl-lang.io/cli/pkg/version.version=v${version}"
   ];
 
-  nativeBuildInputs = [ makeWrapper installShellFiles ];
-  buildInputs = [ kclvm kclvm_cli ];
+  nativeBuildInputs = [ makeWrapper installShellFiles ] ++ (
+    lib.optionals stdenv.isDarwin [ darwin.cctools ]
```

Comment:
`darwin.cctools` moved to `pkgs/by-name/cctools` in staging and staging-next so this is created eval errors.
https://gist.github.com/GrahamcOfBorg/cc66a857ef768344f881acbbaacc32bf which is not a big deal, i'm looking at fixing it. however, isn't cctools already part of the stdenv? Perhaps i'm missing something but it seems like all of the `darwin.cctools` can just be removed and not effect the build.

Reply:
builds fine on x64 darwin removing the cctools from inputs

https://github.com/NixOS/nixpkgs/pull/330118

Reply:
@paparodeo Let me test it, I'll be right back at you, if that's the case. I'll open a separate PR.